crates/mipsy_interactive/src/interactive/commands/breakpoint.rs

```diff
@@ -79,14 +87,16 @@ fn get_long_help() -> String {
         "help breakpoint".bold(),
         "<subcommand>".purple().bold(),
         "<subcommand>".purple(),
+        "temporary".purple(),
+        "ignore".purple(),
     )
 }
 
-fn breakpoint_insert(state: &mut State, label: &str, args: &[String], remove: bool) -> Result<String, CommandError> {
+fn breakpoint_insert(state: &mut State, label: &str, mut args: &[String], remove: bool) -> Result<String, CommandError> {
```

Comment:
Is a temporary breakpoint created with
`breakpoint insert temporary <address>`
or
`breakpoint temporary <address>`
This help says to do both?
`breakpoint temporary temporary <address>`

Reply:
all three are technically valid, I allowed `breakpoint temporary` as a shortcut for `breakpoint insert temporary` and due to quirks in the parsing logic `breakpoint temporary temporary` is valid syntax for creating a temporary breakpoint
I'll try and remove the validity of `breakpoint temporary temporary`

Reply:
If `breakpoint temporary` is valid. I don't see the need for `breakpoint insert temporary`.

But the main issue is that the help text makes it sound like you must use `breakpoint temporary temporary`

Reply:
Yeah I decided it was neatest to only allow `breakpoint temporary` and updated help text to reflect this